etl/steps/__init__.py

```diff
@@ -97,11 +97,33 @@ def to_dependency_order(
 
 
 def load_dag(filename: Union[str, Path] = paths.DAG_FILE) -> Dict[str, Any]:
-    with open(str(filename)) as istream:
-        dag: Dict[str, Any] = yaml.safe_load(istream)
+    return _load_dag(filename, {})
 
-    dag = {node: set(deps) if deps else set() for node, deps in dag["steps"].items()}
-    return dag
+
+def _load_dag(filename: Union[str, Path], prev_dag: Dict[str, Any]):
+    """
+    Recursive helper to 1) load a dag itself, and 2) load any sub-dags
+    included in the dag via 'include' statements
+    """
+    dag_yml = _load_dag_yaml(filename)
+    curr_dag = _parse_dag_yaml(dag_yml)
+    curr_dag.update(prev_dag)
+
+    if "include" in dag_yml:
```

Comment:
Nitpick - this could be written as one-liner
```
for sub_dag_filename in dag_yml.get("include", []):
```

Reply:
👍 Good point